### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Zeeerooo**
New commands documentation.

### CODE REVIEW

File: adkintunmobile/server/README.md [hunk 1]
```diff
@@ -7,19 +7,36 @@ Adkintun Mobile is an android application project for monitoring QoS on mobile n
...
-```bash
-$ ./run.sh [-u <string : user_name_database> ] [-p <string : password>] [-d <string : database_name>]
-```
+1. Edit the configuration file (`config.py`) with the desired values for the database name, user and password, and also with the data for the admin user in the application.
+2. Build the docker images using the following command
 
+  ```shell
+  $ ./run.sh build
+  ```
+3. After, run the containers , giving as parameters the database name, the username and the password of the database (the same given in the step 1):
```
Comment (camilog) [inline adkintunmobile/server/README.md]: you have an extra space in the comma

File: adkintunmobile/server/README.md [hunk 2]
```diff
@@ -7,19 +7,36 @@ Adkintun Mobile is an android application project for monitoring QoS on mobile n
...
 
-### uwsgi
+### server
 Dockerfile for the server application.
+
+### config.py
+File with the differents settings for the application
+
+### run.sh
+Bash script for deploy, run, start, restart, and stop the application. The usage is:
```
Comment (camilog) [inline adkintunmobile/server/README.md]: s/for/to

File: adkintunmobile/server/README.md [hunk 3]
```diff
@@ -7,19 +7,36 @@ Adkintun Mobile is an android application project for monitoring QoS on mobile n
...
+
+
+```bash
+# For build, start, restart, stop, delete, upgrade
```
Comment (camilog) [inline adkintunmobile/server/README.md]: s/for/to

File: adkintunmobile/server/README.md [hunk 4]
```diff
@@ -7,19 +7,36 @@ Adkintun Mobile is an android application project for monitoring QoS on mobile n
...
+$ ./run.sh build | start | restart | stop | delete | upgrade_app
+
+# For run
```
Comment (camilog) [inline adkintunmobile/server/README.md]: s/for/to

Comment (fmontoto) [reply adkintunmobile/server/README.md]: Wouldn't be more consistent `upgrade` instead of `upgrade_app`

**Comment by fmontoto** [comment]
LGTM after @camilog suggestions

File: adkintunmobile/server/README.md (hunk 4 above)
Comment (Zeeerooo) [reply adkintunmobile/server/README.md]: Done. What i have to do with the reviews?

**Comment by camilog** [comment]
LGTM

**Pull request merged**